### PR TITLE
14.0 mig hr expense petty cash

### PR DESCRIPTION
**Pull request by modnoting**

### CODE REVIEW

**Comment by Saran440** [comment]
- Please, squash your commit `[MIG] hr_expense_petty_cash: Migration to 14.0` into 1
- This PR should Migration only (Don't add your feature `Group by` in this PR)

**Comment by bluemix** [comment]
I hope this will be merged soon...

File: hr_expense_petty_cash/views/petty_cash_views.xml [hunk 1]
```diff
@@ -65,7 +65,30 @@
         <field name="model">petty.cash</field>
         <field name="arch" type="xml">
             <search string="Petty Cash">
-                <group expand="0" string="Group By" />
+                <field
```
Comment (Saran440) [inline hr_expense_petty_cash/views/petty_cash_views.xml:68]: Please delete it, Standard with petty cash version13 is not group by.
I think we should add new search group by on PR https://github.com/OCA/hr/pull/924 and help improve it for version 13 and 14

**Review by Saran440 — COMMENTED** [review]
@bluemix please, you review and approve it.

**Comment by kittiu** [comment]
Thanks for your contribution. @modnoting are you still working on this?

**Comment by Saran440** [comment]
move on https://github.com/OCA/hr-expense/pull/66

**Pull request closed without merging**